File: src/main/java/dk/cngroup/wishlist/JacksonConfig.java
package dk.cngroup.wishlist;

import org.openapitools.jackson.nullable.JsonNullableJackson3Module;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import tools.jackson.databind.JacksonModule;

@Configuration
public class JacksonConfig {

    @Bean
    public JacksonModule jsonNullableModule() {
        return new JsonNullableJackson3Module();
    }
}

File: src/main/kotlin/dk/cngroup/wishlist/JacksonConfig.kt
package dk.cngroup.wishlist

import org.openapitools.jackson.nullable.JsonNullableJackson3Module
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import tools.jackson.databind.JacksonModule

@Configuration
class JacksonConfig {

    @Bean
    fun jsonNullableModule(): JacksonModule = JsonNullableJackson3Module()
}
